envs/monkey_zoo/blackbox/config_templates/powershell_cached_credentials.py

```diff
@@ -0,0 +1,21 @@
+from copy import copy
+
+from envs.monkey_zoo.blackbox.config_templates.base_template import BaseTemplate
+from envs.monkey_zoo.blackbox.config_templates.config_template import ConfigTemplate
+
+
+class PowerShellCachedCredentials(ConfigTemplate):
```

Comment:
The name is misleading, cached powershell credentials are stored in windows credential manager. Based on the name of the test one would presume that monkey will steal powershell credentials from credential manager and will use them to login, when in fact this is testing interactive logon without credentials/ current credential reuse.